.drone.yml

```diff
@@ -88,11 +88,20 @@ pipeline:
             matrix:
                 BUILD: AppImage
 
+    Debian:
+        image: nextcloud-drone-debian:latest
```

Comment:
docker tries to pull this image form hub.docker.com... 

Can you create a PR against https://github.com/nextcloud/docker-ci (just put the file in the Dockerfile in the client folder)

Reply:
I have created the PR agains the ```docker-ci``` repository, and also fixed ```.drone.yml``` here assuming that the tag will be ```client-debian-ci-1```.

Please, also note that to be able to sign the source packages, the signing keys should be made available to the build script. The key is contained in an encrypted form in ```admin/linux/debian/signing-key.txt.enc``` (there is also an ```oscrc.enc``` file for the OpenSUSE build server used to build the Debian packages). These files are encrypted using a key/iv pair that should be communicated to the build script, which currently assumes that they are in the environment variables DEBIAN_SECRET_KEY and DEBIAN_SECRET_IV. I found several different ways to store such secrets in the repository for Drone, but could not test any of them. 

Could you find the proper way to store these values? How can I send these values to you in a secure way?